scripts/nbconverted/4_similarity_analysis.py

```diff
@@ -0,0 +1,173 @@
+
+# coding: utf-8
+
+# # Similarity analysis
+# 
+# We want to determine if the different batch simulated data is able to capture the biological signal that is present in the original data:  How much of the real input data is captured in the simulated batch data?
```

Comment:
curious. why svcca?

Reply:
This was a suggestion from Casey to allow us to compare the representation of the input data without batches vs some number of batches.  I definitely need to look into other possible options.